schema-generator/src/main/java/com/github/jrcodeza/schema/generator/OpenAPIGenerator.java

```diff
@@ -63,27 +68,39 @@
     private final List<OperationInterceptor> operationInterceptors;
     private final List<RequestBodyInterceptor> requestBodyInterceptors;
 
+    private AtomicReference<OperationFilter> operationFilter;
+    private AtomicReference<OperationParameterFilter> operationParameterFilter;
+    private AtomicReference<SchemaFieldFilter> schemaFieldFilter;
+
     private final List<Header> globalHeaders;
 
     public OpenAPIGenerator(List<String> modelPackages, List<String> controllerBasePackages, Info info) {
-        this(modelPackages, controllerBasePackages, info, new ArrayList<>(), new ArrayList<>(), new ArrayList<>(), new ArrayList<>(), new ArrayList<>());
+        this(modelPackages, controllerBasePackages, info, new ArrayList<>(), new ArrayList<>(), new ArrayList<>(), new ArrayList<>(), new ArrayList<>(), null, null, null);
     }
 
     public OpenAPIGenerator(List<String> modelPackages, List<String> controllerBasePackages, Info info,
                             List<SchemaInterceptor> schemaInterceptors,
                             List<SchemaFieldInterceptor> schemaFieldInterceptors,
                             List<OperationParameterInterceptor> operationParameterInterceptors,
                             List<OperationInterceptor> operationInterceptors,
-                            List<RequestBodyInterceptor> requestBodyInterceptors) {
+                            List<RequestBodyInterceptor> requestBodyInterceptors,
+                            OperationFilter operationFilter,
```

Comment:
I would add another constructor. So this one would remain as is and new one would have thee 3 filters. Then the smaller one can call the complete one where these 3 additional filters would be null. It's for sake of backwards compatibility

Reply:
yes you're right, I didn't think about backwards compatibility. I added a commit to fix that.